docs/software/apptainer.md

```diff
@@ -0,0 +1,87 @@
+# Apptainer
+
+Apptainer and other container runtimes have many uses; Mainly, but not limited to, isolated, portable, and reproducible software execution and development. For running containers on HPC systems we offer and use Apptainer ( formerly Singularity ). Other container runtimes like Docker require users to have escalated system priveleges, and for most thats typically not an issue, for shared multi-user systems letting any and all users have escalated privileges can pose a security risk. Luckily, apptainer allows users to run unprivileged containers in user-space inluding Docker containers and any OCI compatible containers.
+
+## Module System
+
+On HPC systems, we provide apptainer, as an environment module. Everytime you want to use apptainer you must first run: `module load apptainer`.
```

Comment:
First sentence could be simplified: `HPC@UCD provides apptainer as an environment module.`